src/azure/com/microsoftopentechnologies/intellij/wizards/WizardCacheManager.java

```diff
@@ -475,11 +475,12 @@ public static void cachePublishData(File publishSettingsFile, PublishData publis
 					if (services != null) {
 						for (StorageService strgService : services) {
 							List<URI> endpoints = strgService.getStorageAccountProperties().getEndpoints();
-							if (endpoints.get(0).toString().startsWith("https://")) {
-								endpoints.set(0, URI.create(endpoints.get(0).toString().replaceFirst("https://", "http://")));
-								endpoints.set(1, URI.create(endpoints.get(1).toString().replaceFirst("https://", "http://")));
-								endpoints.set(2, URI.create(endpoints.get(2).toString().replaceFirst("https://", "http://")));
-							}
+                            for (int i = 0; i < endpoints.size(); i++) {
+                                String uri = endpoints.get(i).toString();
+                                if (uri.startsWith("https://")) {
+                                    endpoints.set(i, URI.create(uri.replaceFirst("https://", "http://")));
+                                }
+                            }
```

Comment:
This is not related to this bug fix but I am wondering why we are removing SSL from the endpoint URL?